devops/iam-did-auth-proxy-prod/values.yaml

```
@@ -0,0 +1,284 @@
+# Default values for did-auth-proxy-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+did-auth-proxy-helm:
+  enabled: true
+  replicaCount: 1
+
+  image:
+    repository: 984870885661.dkr.ecr.us-west-2.amazonaws.com/did-auth-proxy
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "latest"
+
+  imagePullSecrets: []
+
+  nameOverride: "did-auth-proxy-helm"
+  fullnameOverride: "did-auth-proxy-helm"
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    type: ClusterIP
+    port: 80
+
+  ingress:
+    enabled: false
+
+  config:
+    enabled: false
+    secretRefName: {}
+    configRefName: {}
+
+  appValues:
+    PORT: 80
+    CORS_MAX_AGE: 60
+    CORS_ORIGIN: https://gp4btc-miner.energyweb.org,https://gp4btc-explorer.energyweb.org,http://localhost:4200
+    CACHE_SERVER_LOGIN_PRVKEY: eab5e5ccb983fad7bf7f5cb6b475a7aea95eff0c6523291b0c0ae38b5855459c
+    JWT_SECRET: asecretstring
+    JWT_ACCESS_TTL: 3600
+    JWT_REFRESH_TTL: 86400
+    DID_REGISTRY_ADDRESS: 0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af
+    ENS_REGISTRY_ADDRESS: 0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac
+    ENS_RESOLVER_ADDRESS: 0xcf72f16Ab886776232bea2fcf3689761a0b74EfE
+    INCLUDE_ALL_ROLES: false
+    SWAGGER_PATH: api-auth
+    BLOCKNUM_AUTH_ENABLED: true
```

We don't want `BLOCKNUM_AUTH` in `prod`
```suggestion
    BLOCKNUM_AUTH_ENABLED: false
```